### ISSUE TITLE
Readable cassette payloads for VCR

### ISSUE THREAD

**Issue by abhinav** [uber/tchannel-python#361]
Recorded Thrift payloads should be readable. With thriftrw, we should be able to do it fairly easily. We'll have to figure out a migration path for existing cassettes that are using base64 blobs.

**Comment by breerly** [comment]
Let's wait until we drop Apache Thrift support.

**Comment by abhinav** [comment]
Yeah. When we add this, it'll be to the mash release branch or after that's released.
